src/__experimental__/components/date/date.component.js

```diff
@@ -285,21 +317,24 @@ class BaseDateInput extends React.Component {
   }
 
   render() {
-    const { minDate, maxDate, ...inputProps } = this.props;
+    const {
+      minDate, maxDate, isDateRange, ...inputProps
```

Comment:
Should we add `isDateRange` to `BaseDateInput.propTypes` in this file?

Reply:
I think I might try and refactor this as I don't think it's a great solution having `Date` know about `DateRange`

Reply:
On further review I'm going to leave this as is and add a propType. It works as a hot-fix and then I'll raise another ticket to refactor how the validation is applied  by implementing `context` that can be overriden when part of a `DateRange`

Reply:
Thanks for adding the `propType` in 38d0e2a :white_check_mark: